.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Please make sure to **read the following list** before creating a new issue:
+
+* Do you have a question on how to use vis.js? Please don't file an issue but ask your question e.g on [stackoverflow](https://stackoverflow.com/questions/tagged/vis.js) by using the tag 'vis.js' We will close use-specific questions!
```

Comment:
Maybe better: This issue tracker is **not** supposed to be used for questions on how to use visjs. It is intended to be used for bug reports! In case you face yourself with a usage question, then post your issue on StackOverflow tagged with `vis.js`.